modules/claims_api/app/sidekiq/claims_api/poa_updater.rb

```diff
@@ -28,6 +28,8 @@ def perform(power_of_attorney_id) # rubocop:disable Metrics/MethodLength
 
         ClaimsApi::Logger.log('poa', poa_id: poa_form.id, detail: 'BIRLS Success')
 
+        ClaimsApi::VANotifyJob.perform_async(poa_form.id, icn_for_vanotify, rep) if icn_for_vanotify.present?
```

Comment:
https://github.com/department-of-veterans-affairs/vets-api/pull/18687/files#r1781912411 sames.